nautobot/ipam/templates/ipam/vlan.html

```diff
@@ -140,12 +91,9 @@ <h1>{% block title %}VLAN {{ object.display }}{% endblock %}</h1>
                     </tr>
                 </table>
             </div>
-            {% include 'inc/custom_fields_panel.html' %}
-            {% include 'inc/relationships_panel.html' %}
-            {% include 'extras/inc/tags_panel.html' with tags=object.tags.all url='ipam:vlan_list' %}
-            {% plugin_left_page object %}
-        </div>
-        <div class="col-md-8">
+{% endblock content_left_page %}
+
+{% block content_right_page %}
```

Comment:
Consider `content_full_width_page`? Though the left column is fairly long so this may not be ideal.

![image](https://user-images.githubusercontent.com/5603551/127335703-10bfd37d-ac07-405c-9026-f3e82a5be0b6.png)

Reply:
![Cursor_and_vlan1__1__-_Nautobot](https://user-images.githubusercontent.com/138052/127353477-195cf392-5ae9-4b73-abee-9b04ffcc526a.png)

Looks good to me.